src/main.ts

```diff
@@ -28,17 +28,24 @@ const getInputs = () => {
   const GITHUB_TOKEN: string = core.getInput('github-token', { required: true });
   const BRANCH_IGNORE_PATTERN: string = core.getInput('skip-branches', { required: false }) || '';
   const SKIP_COMMENTS: string = core.getInput('skip-comments', { required: false }) || 'false';
+  const PR_THRESHOLD: string = core.getInput('pr-threshold', { required: false }) || '';
```

Comment:
@rheaditi Feel free to suggest a better and more relatable name

Reply:
This is good :+1: